content/posts/mfa.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "2-Factor Authentication!"
+excerpt: "Easily secure your Aptible accounts."
+author_name: Thomas Orozco
+author_email: thomas@aptible.com
+author_id: thomas
+posted: 2016-05-19
+section: Blog
+posts: true
+---
+We're happy to announce that two-factor authentication (2FA) is available for all users and account types in the Aptible dashboard and CLI! Multifactor authentication is a best practice that adds an additional layer of security on top of the normal username and password you use to verify your identity. You can enable it in your Aptible user settings.
+
+## How does it work?
+
+Aptible 2-factor authentication implements the Time-based One-time Password (TOTP) algorithm specified in RFC 6238. We currently support the virtual token form factor - [Google Authenticator][0] is an excellent, free app. We do not currently support SMS or hardware tokens.
```

Comment:
The sentence on Google Authenticator read a little weird to me (as in it felt like it was missing a word or two at the end). Might have to do with not being a native speaker, but I would perhaps have used e.g.:

> Google Authenticator is an excellent free app that implements it.